engines/sg.c

```diff
@@ -472,7 +472,6 @@ static enum fio_q_status fio_sgio_rw_doio(struct thread_data *td,
 				break;
 
 			if (io_u_sync_complete(td, __io_u)) {
-				ret = -1;
```

Comment:
First I thought this might be a legit bug, but actually looks fine. You still need to remove the braces, though.

Reply:
Just upload a new version with the braces.